src/mainwindow.cpp

```diff
@@ -24,6 +24,8 @@ MainWindow::MainWindow(QWidget *parent) :
     connect(app->getAction(KiwixApp::AboutAction), &QAction::triggered,
             mp_about, &QDialog::show);
     connect(app, &KiwixApp::currentTitleChanged, this, [=](const QString& title) {
+        if (title.startsWith("zim://"))
+            return;
```

Comment:
Maybe set the title to "Library - Kiwix" in this case. So we avoid to keep the previous (wrong) title of the previous page.